tube/etl/indexers/base/translator.py

```diff
@@ -72,8 +72,18 @@ def write(self, df):
     def get_props_from_data_row(self, df, props, to_tuple=False):
         if df.isEmpty():
             return df.mapValues(get_props_empty_values(props))
-        names = {p.src: p.id for p in props}
-        values = {p.src: {m.original: m.final for m in p.value_mappings} for p in props}
+        names = {}
+        values = {}
```

Comment:
a comment with an example of input and associated `names` `values` would be useful to understand this code